examples/file-and-cloudinary-upload/app/routes/local-upload.tsx

```diff
@@ -0,0 +1,52 @@
+import {
+  Form,
+  unstable_createFileUploadHandler,
+  unstable_parseMultipartFormData,
+  useActionData,
+  json
+} from "remix";
+import type { ActionFunction } from "remix";
+
+type ActionData = {
+  errorMsg?: string;
+  imgSrc?: string;
+};
+
+export const action: ActionFunction = async ({ request }) => {
+  const uploadHandler = unstable_createFileUploadHandler({
+    directory: "public",
+    maxFileSize: 30000
+  });
+  const formData = await unstable_parseMultipartFormData(
+    request,
+    uploadHandler
+  );
+  const image = formData.get("img");
+  if (!image) {
+    return json({
+      error: "something wrong"
+    });
+  }
+  return json({
+    imgSrc: image.name
```

Comment:
Property 'name' does not exist on type 'FormDataEntryValue'.
https://stackoverflow.com/questions/71090990/typescript-property-name-does-not-exist-on-type-formdataentryvalue